extensions/datavirtualization/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "datavirtualization",
   "displayName": "%title.datavirtualization%",
   "description": "%config.extensionDescription%",
-  "version": "1.13.2",
+  "version": "1.14.0",
   "publisher": "Microsoft",
   "icon": "resources/extension.png",
   "aiKey": "29a207bb14f84905966a8f22524cb730-25407f35-11b6-4d4e-8114-ab9e843cb52f-7380",
   "engines": {
     "vscode": "^1.48.0",
-    "azdata": "^1.22.0"
```

Comment:
I wasn't sure if these needed to be updated, but thought it's safer since we're only testing on the latest version

Reply:
I'd remove this unless there's actually a compatibility issues that we know of (which there shouldn't be since the changes aren't functional in nature)

Reply:
undid those two changes